image/cli/mascli/functions/gitops_cp4d

```diff
@@ -238,21 +245,68 @@ function gitops_cp4d() {
   TEMP_DIR=$CURRENT_DIR/tmp-cp4d
   mkdir -p $TEMP_DIR
 
+  if [ -z $GIT_SSH ]; then
+    export GIT_SSH="false"
+  fi
+
 
-  # We want to use CP4D 4.8 but we need to wait for it to be ready in automation and product
-  # In order to not block use we will just install using ansible-devops
   AVP_TYPE=aws  # Support for IBM will be added later
   sm_login
-  export IBM_ENTITLEMENT_KEY_FILE=$TEMP_DIR/entitlement.yaml
-  export SECRET_IBM_ENTITLEMENT=${ACCOUNT_ID}${SECRETS_KEY_SEPERATOR}${CLUSTER_ID}${SECRETS_KEY_SEPERATOR}ibm_entitlement
-  sm_get_secret_file $SECRET_IBM_ENTITLEMENT $IBM_ENTITLEMENT_KEY_FILE
 
-  export IBM_ENTITLEMENT_KEY=$(jq -r .entitlement_key $IBM_ENTITLEMENT_KEY_FILE | base64 -d ; echo)
-  export ROLE_NAME=cp4d
-  ansible-playbook ibm.mas_devops.run_role
+  export SECRET_KEY_IBM_ENTITLEMENT=${ACCOUNT_ID}${SECRETS_KEY_SEPERATOR}${CLUSTER_ID}${SECRETS_KEY_SEPERATOR}ibm_entitlement#image_pull_secret_b64
+  export SECRET_KEY_NAMESPACE_SCOPE_CHANNEL=${ACCOUNT_ID}${SECRETS_KEY_SEPERATOR}${CLUSTER_ID}${SECRETS_KEY_SEPERATOR}${MAS_INSTANCE_ID}${SECRETS_KEY_SEPERATOR}namespace_scope_channel#namespace_scope_channel
+  export SECRET_KEY_LICENSING_CHANNEL=${ACCOUNT_ID}${SECRETS_KEY_SEPERATOR}${CLUSTER_ID}${SECRETS_KEY_SEPERATOR}${MAS_INSTANCE_ID}${SECRETS_KEY_SEPERATOR}licensing_channel#licensing_channel
+  export SECRET_KEY_LICENSING_VERSION=${ACCOUNT_ID}${SECRETS_KEY_SEPERATOR}${CLUSTER_ID}${SECRETS_KEY_SEPERATOR}${MAS_INSTANCE_ID}${SECRETS_KEY_SEPERATOR}licensing_version#licensing_version
+  export SECRET_KEY_CPFS_CHANNEL=${ACCOUNT_ID}${SECRETS_KEY_SEPERATOR}${CLUSTER_ID}${SECRETS_KEY_SEPERATOR}${MAS_INSTANCE_ID}${SECRETS_KEY_SEPERATOR}cpfs_channel#cpfs_channel
+  export SECRET_KEY_CPD_PLATFORM_CHANNEL=${ACCOUNT_ID}${SECRETS_KEY_SEPERATOR}${CLUSTER_ID}${SECRETS_KEY_SEPERATOR}${MAS_INSTANCE_ID}${SECRETS_KEY_SEPERATOR}cpd_platform_channel#cpd_platform_channel
+
+
+  # Default settings
+  export CPD_CS_CONTROL_NAMESPACE="cs-control"
+  export CPD_ADMIN_LOGIN_SA="mas-cpd-cluster-admin-sa"
+  export CPD_PLATFORM_CR_NAME="ibmcpd-cr"
+  export CPD_IAM_INTEGRATION=false
+  export CPFS_SIZE="small"
+  export CPD_SCALE_CONFIG="medium"
```

Comment:
We would want to have these last two configurable:
```
  export CPFS_SIZE="small"
  export CPD_SCALE_CONFIG="medium"
```

So have the defaults but let those be overwritten if are passed in. these would also need to be exposed on the task/pipeline